### PR TITLE
Update install.sh to support Xcode 7.3

### PR DESCRIPTION
**Pull request by ebaker355**
The script will now check whether the DVTIbeamCursor.tiff exists and update that as usual. If it does not exist, it will check if Assets.car (Xcode 7.3) exists and update that in the same manner.

Please review and test the script thoroughly; I am not an experienced shell script developer. :)

### CODE REVIEW

**Comment by maxsz** [comment]
I think you need to consider the copyright. Distributing the whole Assets.car file, including all images, will probably infringe Apple's copyright. We might need to come up with a solution that only patches the Assets.car file... :(

**Comment by ebaker355** [comment]
@maxsz That's a good point. Perhaps we can create a binary diff of the changes in the new Assets.car file, then just patch (after backing up) the original file. In case the original Assets.car file ever changes, we could associate the patch to a SHA of the original.

**Comment by ebaker355** [comment]
I am working on the script, it has a few bugs at the moment. I will reopen this PR when they are worked out. Hopefully later today.

**Comment by egold** [comment]
Cool let me know once you get the script working. Sounds pretty awesome.

**Comment by ebaker355** [comment]
Sorry guys, I've made _some_ progress, but I've been very busy at work. I should be able to get it going this weekend.

**Comment by ebaker355** [comment]
Also, since we're going down the "patch" route, I'm thinking maybe we can just patch the old DVTIbeamCursor.tiff file as well, rather than replacing it. That way the install script doesn't have to operate in 2 different modes; the only change in operation will be _which file is being patched_, depending on Xcode version.

**Comment by ebaker355** [comment]
@egold Ok, I have updated the install script and reopened this PR. The script now simply patches Assets.car, rather than replacing it. (We're still replacing the older cursor tiff file, if its found).

The script tries to be smart about detecting which patch file is needed. Right now, there is only one patch file for Xcode 7.3. However, if that file changes in future versions of Xcode, the script can be updated to apply the proper patch file. (I'll let someone else handle updates :) )

As before, please test thoroughly.

**Comment by ebaker355** [comment]
Please also note, the patch file was created using `bsdiff`, which can be installed via homebrew. The patch is applied using `bspatch`, which seems to ship with OS X (or Xcode; I'm not certain how it gets installed). The install.sh script only requires `bspatch` to be present.

File: README.md [hunk 1]
```diff
@@ -29,9 +29,13 @@ curl -L https://raw.githubusercontent.com/egold/better-xcode-ibeam-cursor/master
 
 ###### For Xcode 7.3 and later:
 
-1. Clone this repository
+1. Clone this repository (the lines below assume you've cloned to your home directory)
 2. Create a backup of `/Applications/Xcode.app/Contents/SharedFrameworks/DVTKit.framework/Resources/Assets.car`
-3. Copy (`sudo cp`) the Assets.car file to `/Applications/Xcode.app/Contents/SharedFrameworks/DVTKit.framework/Resources/Assets.car`
+3. Patch the Assets.car file with the appropriate patch:
+```
+cd /Applications/Xcode.app/Contents/SharedFrameworks/DVTKit.framework/Resources/Assets.car
```
Comment (mikelow) [inline README.md]: Did you mean `cd /Applications/Xcode.app/Contents/SharedFrameworks/DVTKit.framework/Resources` here? Thanks for the patch, btw!

Comment (ebaker355) [inline README.md]: @mikelow Yep! Good catch! Thanks.

**Comment by manas-chaudhari** [comment]
Please update the README also

**Pull request merged**